tests/test_graphqa.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `graphqa` package."""
+
+import pytest
```

Comment:
Some of your unit tests using `unittest` while here using `pytest`, any of them is fine but better stick with either `unittest` or `pytest.`

Reply:
Agreed. We will also need more test scripts to test other functions.